src/testPane.ts

```diff
@@ -571,9 +571,9 @@ function launchConfigExists(launchJsonPath: string): boolean {
   // this will check if launch.json has
   // the "VectorCAST Harness Debug" configuration defined
   let returnValue = false;
-  const existingJSON: any = loadLaunchFile(launchJsonPath);
-  if (existingJSON.configurations) {
-    for (const existingConfig of existingJSON.configurations) {
+  const existingJSONdata: any = loadLaunchFile(launchJsonPath);
+  if (existingJSONdata) {
```

Comment:
What if `existingJSONdata.jsonData` doesn't have `configurations`?

Should it be: `if (existingJSONdata and existingJSONdata.jsonData.configurations)`?

Reply:
Good catch, thanks.   Fix pushed - see comment